appinventor/buildserver/src/com/google/appinventor/buildserver/tasks/android/RunApkSigner.java

```diff
@@ -33,12 +33,15 @@ public TaskResult execute(AndroidCompilerContext context) {
         "-ks", context.getKeystoreFilePath(),
         "-ks-key-alias", "AndroidKey",
         "-ks-pass", "pass:android",
+        // "-key-pass", "pass:android",
         context.getPaths().getDeployFile().getAbsolutePath()
     };
 
     if (!Execution.execute(null, apksignerCommandLine,
         System.out, System.err)) {
-      TaskResult.generateError("Error while running ZipAligned tool");
+      context.getReporter().warn("We could not sign your app. In case you are using a custom keystore, please make " +
+        "sure its password is set to 'android', and the key is set to 'androidkey'.");
+      return TaskResult.generateError("Error while running ApkSigner tool");
```

Comment:
I think this is okay for now but for critical errors we need to think about how to internationalize them.